### DISCUSSION TITLE
Aide

### DISCUSSION THREAD

**Discussion by TomHitch** [iptv-org/community#550] [Q&A]
Quelqu'un peut m'aider à démarrer je comprends rien 
merci

**Comment by ghost** [comment]
je voulais tout simplement l'application smarters iptv pro je ne veux pas la liste je veux juste l'application j'ai deja mon code

**Comment by BellezaEmporium** [comment]
> Quelqu'un peut m'aider à démarrer je comprends rien merci

Il vous faut un lecteur vidéo (type VLC, MPC-HC ou autre), puis récupérer le lien présent dans "Playlists", le copier coller dans le lecteur vidéo de votre choix (sur VLC c'est "ouvrir un flux réseau", sur MPC-HC c'est ouvrir puis automatiquement coller sur "fichier ou flux réseau")

**Comment by TomHitch** [comment]
Merci maintenant il manque plus que les chaines française et voir les chaines de sports en bonus

**Reply by sandrinejj** [reply]
dans mon cas les chaînes françaises ne fonctionne pas .
